pages/shoppingCart.js

```diff
@@ -1,18 +1,17 @@
+import Header from "@/components/mainHeader/header";
+import BackButton from "@/components/backButton/backButton";
 import styled from "styled-components";
-import { useRouter } from "next/router";
-import { useEffect, useState } from "react";
+import { useState } from "react";
 import useLocalStorageState from "use-local-storage-state";
 import Image from "next/image";
+import cartpic from "../public/kelvin-moquete.jpg";
+import Lottie from "lottie-react";
+import gifsuccess from "../public/animation_success.json";
 
-export default function ShoppingCart({ ticket }) {
-  const router = useRouter();
+export default function ShoppingCart() {
   const [showPopup, setShowPopup] = useState(false);
   const [cartTickets, setCartTickets] = useLocalStorageState("cartTicket", []);
 
-  const goBack = () => {
-    router.push("/");
-  };
-
   const handleDeleteTicket = (ticketToDelete) => {
     const updatedCartTickets = cartTickets.filter(
       (ticket) => ticket !== ticketToDelete
```

Comment:
This is not too elegant because it's prone to errors. You are comparing objects. Objects are compared by reference, not their "content". It would be more prudent to use an id for comparison.